python/requirements_ml_docker.txt

```diff
@@ -8,3 +8,10 @@ tblib
 -f https://download.pytorch.org/whl/torch_stable.html
 torch==1.9.0+cu111
 torchvision==0.10.0+cu111
+
+-f https://data.pyg.org/whl/torch-1.9.0+cu111.html
```

Comment:
curious, what is this for?

Reply:
These are required dependencies for pytorch geometric